### PR TITLE
configure.ac: Drop auto-recognition of GTK library, default to GTK-3.

### PR DESCRIPTION
**Pull request by sunweaver**

### CODE REVIEW

**Comment by sunweaver** [comment]
@Ionic: We really need to default to GTK-3 and an auto-recognition feature is not required.

File: configure.ac [hunk 1]
```diff
@@ -32,42 +32,37 @@ PKG_CHECK_MODULES(ARCTICA_GREETER, [
...
 with_gtk4=$enable_gtk4
 
 AS_IF([test "x$enable_gtk4" != 'xno'],
       [PKG_CHECK_MODULES([GTK],
-                         [gtk4
-                          gtk4-x11],
+                         [gtk4 gtk4-x11],
                          [gtk4_modules=yes],
                          [gtk4_modules=no])
        AC_MSG_CHECKING([for GTK 4 libraries])
```
Comment (Ionic) [inline configure.ac]: Will change this to `GTK-4` before merging if you want to have it that way for consistency.

File: configure.ac [hunk 2]
```diff
@@ -32,42 +32,37 @@ PKG_CHECK_MODULES(ARCTICA_GREETER, [
...
        AC_MSG_RESULT([$gtk4_modules])
        AS_IF([test "x$gtk4_modules" = 'xno'],
-             [AS_IF([test "x$enable_gtk4" != 'xauto'],
-                    [AC_MSG_ERROR([requested GTK 4, but not found])],
-                    [with_gtk4=no])],
+             [AC_MSG_ERROR([requested GTK 4, but not found])],
```
Comment (Ionic) [inline configure.ac]: Same here.

File: configure.ac [hunk 3]
```diff
@@ -32,42 +32,37 @@ PKG_CHECK_MODULES(ARCTICA_GREETER, [
...
                          [gtk3_modules=yes],
                          [gtk3_modules=no])
        AC_MSG_CHECKING([for GTK+ 3 libraries])
        AC_MSG_RESULT([$gtk3_modules])
        AS_IF([test "x$gtk3_modules" = 'xno'],
-             [AS_IF([test "x$enable_gtk4" = 'xauto'],
-                    [AC_MSG_ERROR([fallback to GTK+ 3, but not found])],
-                    [AC_MSG_ERROR([requested GTK+ 3, but not found])])])])
+             [AC_MSG_ERROR([requested GTK+ 3, but not found])],
+             [])])
```
Comment (Ionic) [inline configure.ac]: Will drop this empty else branch if we're not using it anyway.

**Review by Ionic — DISMISSED** [review]
Looks good in general, will force-push a few changes before merging this.

**Review by Ionic — APPROVED** [review]
Amended as discussed in the previous review.

**Pull request merged**